test/lib/ses_test.exs

```diff
@@ -171,7 +171,7 @@ defmodule ExAws.SESTest do
         )
 
       assert operation.http_method == :put
-      assert operation.path == "/v2/email/contact-lists/#{@list_name}/contacts/#{email}"
+      assert operation.path == "/v2/email/contact-lists/#{@list_name}/contacts/#{ExAws.Request.Url.uri_encode(email)}"
```

Comment:
Would you please set specific expectation (with a string literal)?

Calling the same function as in production code makes test less relaible.

Reply:
Sure thing 👍